client/components/domains/contact-details-form-fields/custom-form-fieldsets/constants.js

```diff
@@ -4,8 +4,8 @@ export const CHECKOUT_UK_ADDRESS_FORMAT_COUNTRY_CODES = [ 'GB', 'IE' ];
 // a localized list of states from the backend:
 // AU, BE, BR, CN, ES, IN, IT, JP, MX
 export const CHECKOUT_EU_ADDRESS_FORMAT_COUNTRY_CODES = [
-	'AR', 'AT', 'BA', 'BG', 'CH', 'CL', 'CZ', 'DE',
-	'DK', 'EE', 'FI', 'FR', 'HU', 'IS', 'IL', 'LU', 'MC',
+	'AR', 'AT', 'BA', 'BG', 'CH', 'CL', 'CZ', 'DE', 'DK',
+	'EE', 'ES', 'FI', 'FR', 'HU', 'IS', 'IL', 'LU', 'MC',
```

Comment:
Adding the country code won't work on its own. To prevent submission errors, we [check if the backend returns (and expects) state values](https://github.com/Automattic/wp-calypso/blob/9f3151d/client/components/domains/contact-details-form-fields/custom-form-fieldsets/region-address-fieldsets.jsx#L48) as well.

As mentioned, we'd probably need to remove these states from the list.